.github/scripts/nm-run-lm-eval-gsm-hf-baseline

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+# We can use this script to compute baseline accuracy on GSM for transformers.
```

Comment:
It would be nice to have more than gsm8k here, which is what was originally implemented in lm_eval_compare_hf_vs_vllm.py, but this is fine for now.

Reply:
yep agree. just trying to get the structure in place